asapdiscovery-data/asapdiscovery/data/schema_v2/ligand.py

```diff
@@ -189,11 +204,19 @@ def from_smiles(cls, smiles: str, **kwargs) -> "Ligand":
     @property
     def smiles(self) -> str:
         """
-        Get the SMILES string for the ligand
+        Get the canonical isomeric SMILES string for the ligand
         """
         mol = self.to_oemol()
         return oemol_to_smiles(mol)
```

Comment:
I would explicitly set `isomeric=True` in case the default value gets accidentally changed in the function.